mergebot-config.json

```
@@ -3,7 +3,7 @@
     "enabled": true,
     "interactive": false,
     "fix_version_map": {
-      "master": "Kommander 1.4.1",
+      "master": "Kommander 1.5.0",
```

```suggestion
      "master": "Kommander 1.4.3",
```

If we want to keep developing on master for 1.4, then we just need to be more diligent about updating this config whenever we release a 1.4.x patch. (well, same goes if we use 1.4.x branch, we still need to remember to bump the versions here after each reelase)